hack/lib/build/constants.sh

```diff
@@ -58,24 +58,6 @@ readonly OPENSHIFT_BINARY_SYMLINKS=(
   openshift-manage-dockerfile
   openshift-extract-image-content
   origin
-  osc
-  oadm
-  osadm
-  kubectl
-  kubernetes
-  kubelet
-  kube-proxy
-  kube-apiserver
-  kube-controller-manager
-  kube-scheduler
-)
-readonly OPENSHIFT_BINARY_COPY=(
-  oadm
```

Comment:
This is what actually creates the symlink - you can't remove this without breaking the RPM path.  We still need to test that the symlink works.

Reply:
BINARY_COPY means "create a hardlink".

Reply:
> This is what actually creates the symlink - you can't remove this without breaking the RPM path. We still need to test that the symlink works.

Just so I'm understand here.  The RPM is laying down a symlink called `kube-apiserver` and this is behavior we want to keep?

Or are you commenting specifically about the `oadm` bit, in which case we should be able to simply remove it since that binary has long been replaced by `oc adm`?